tests/manage/z_cluster/nodes/test_toleration.py

```diff
@@ -13,6 +18,8 @@
 logger = logging.getLogger(__name__)
 
 
+@skipif_external_mode
```

Comment:
why is this test skipped on ext mode ?? 
IMO the OCP cluster is going to stay intact and also there will be pods running on openshift-storage ns except mons, mgr and osds, I dont see the need to skip this testcase. Can you please recheck once

Reply:
The problem is with the `get_ocs_nodes` function. With external mode, it didn't find the OCS nodes.

Reply:
I agree with Shrivaibavi, except for ceph related pods we still have other pods in openshift-storage ns.
Perhaps we can consider non-ocs nodes to be used in case of external mode.